picfixPy/vibrance.py

```diff
@@ -1,20 +1,54 @@
-def vibrance(input_img, intensity=5, output_img):
-    
+def vibrance(input_img, intensity=0, display = False, output = ''):
+
     '''
-    Adjust the vibrance of an image, given the intensity specificication
+    Increase or decrease the vibrance of an image, given the intensity specificication.
 
     Input
     ============
-    input_img -- string of path for an image file in .png format
-    intensity -- integer in range [0,10] to specificy intensity of vibrance enhancement applied to the image,
-                 defaults to 5.
-    output_img -- string path for output .png image with adjusted vibrance
+
+    img -- string. Path for an image file in .jpg, .jpeg, .png, .tiff format.
```

Comment:
Catch a minor inconsistency here with the function documentation. I am still quite uncomfortable supporting non-png image format as input at this stage, before we code up the R functions. I suggest keeping things simple and stick with .png images for now. Just my two cents.
```suggestion
    input_img -- string. Path for an image file.
```